share/scan.proto

```diff
@@ -133,6 +133,8 @@ message ScanResult {
     ScanProvider Provider = 23;
     int64 Size = 24;
     repeated string Verifiers = 25;
+    ScanErrorCode SignatureVerificationError = 26;
```

Comment:
Can we change lin 135~137 to something like
message SigstoreResult {
    repeated string Verifiers = 1;
    ScanErrorCode SignatureVerificationError = 2;
    string SignatureVerificationTimestamp = 3;
}

message ScanResult {
    ......
    int64 Size = 24;
    SigstoreResult SigResult = 25;
}